src/smartbanner.js

```diff
@@ -70,9 +70,17 @@ function restoreContentPosition() {
 
 export default class SmartBanner {
 
-  constructor() {
-    let parser = new OptionParser();
-    this.options = parser.parse();
+  constructor(options) {
+    // DEPRECATED. Will be removed in a future major version
```

Comment:
Non-API usage will not become deprecated. The encouraged way of usage will still be over meta tags.